src/Extensions/Abstractions/ReplayableLogs/IActivityProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Provides activities</summary>
+	public interface IActivityProvider
+	{
+		/// <summary>
+		/// Create activity instance
+		/// </summary>
+		/// <param name="operationName">The name of the operation</param>
+		/// <param name="replayLogsInCaseOfError">Should activity collect log events to replay them in case of an error</param>
+		/// <returns></returns>
```

Comment:
>returns [](start = 7, length = 7)

Missing information

Reply:
I've added a comment in this file but for some reasons, it's not displayed here, You can see it using this link https://github.com/microsoft/Omex/blob/feature/create-omex-extensions/src/Extensions/Abstractions/ReplayableLogs/IActivityProvider.cs